src/tests/Footer.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { render, fireEvent } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import '@testing-library/jest-dom/extend-expect';
+import App from '../App';
+
+
+describe('footer', () => {
+  it('navigates to ooloo employers page when ooloo.io link is clicked', () => {
+    const history = createMemoryHistory();
+    const { getByText } = render(
+      <Router history={history}>
+        <App />
+      </Router>,
+    );
+
+    const oolooLink = getByText('ooloo.io');
+    expect(oolooLink.getAttribute('href')).toEqual('https://ooloo.io/employers');
+  });
+
+  it('navigates to home page when logo is clicked', () => {
+    const history = createMemoryHistory();
+    const { getByRole } = render(
+      <Router history={history}>
+        <App />
+      </Router>,
+    );
+
+    const logoLink = getByRole('link', { name: /sign\.svg/ });
```

Comment:
That's a great way to get the logo :smile:, never saw that before

Reply:
Honestly, I noticed the text associated with it when testing.  It didn't like what I did originally so the library suggested some accessible elements and their text and that's when I saw that and thought "perfect!"